aria/modeling/service_template.py

```diff
@@ -97,6 +97,34 @@ def outputs(cls):
     def operation_templates(cls):
         return cls.one_to_many_relationship('operation_template', dict_key='name')
 
+    @declared_attr
+    def node_types(cls):
+        return cls.one_to_one_relationship('type', key='node_type_fk', backreference='')
```

Comment:
this solution looks weird. Having to pass blank string in order not to have backref...